addons/auth_signup/controllers/main.py

```diff
@@ -86,6 +93,11 @@ def web_auth_reset_password(self, *args, **kw):
             except Exception as e:
                 qcontext['error'] = str(e)
 
+        elif 'signup_email' in qcontext:
+            user_ids = request.env['res.users'].sudo().search([('partner_id.email', '=', qcontext.get('signup_email'))])
```

Comment:
is it necessary to do `partner_id.email` instead of just `email` ? (it's a related field so I do not think it's necessary)